CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## 38.0
```

Comment:
nit: some of these have periods at the end and some do not. No preference on which standard we should use, but we should be consistent.

same below in xml file.